docs/main/user-guides/upgrades.md

```diff
@@ -99,25 +99,24 @@ Add both of these options, if your chart has both of these enabled.
 
 ## Replicated Storage
 
-This section describes the Replicated Storage upgrade from OpenEBS Umbrella chart 3.x to OpenEBS 4.1.0.
+This section describes the Replicated Storage upgrade from OpenEBS Umbrella chart 3.x to OpenEBS 4.1.1.
 
-1. Start the helm upgrade process with the new chart, i.e. 4.1.0 by using the below command:
+1. Start the helm upgrade process with the new chart, i.e. 4.1.1 by using the below command:
 
 :::caution
-Disable the partial rebuild during the upgrade from specific versions of OpenEBS (3.7.0, 3.8.0, 3.9.0 and 3.10.0) to OpenEBS 4.1.0 to ensure data consistency during upgrade. Input the value `--set mayastor.agents.core.rebuild.partial.enabled=false` in the **helm upgrade** command.
+Upgrades from 3.x to 4.x require the option `--set mayastor.agents.core.rebuild.partial.enabled=false` in the **helm upgrade** command to ensure data consistency during upgrade. Upgrades from 4.x onwards to newer version do not require it.
 
 This applies to the **kubectl mayastor upgrade** command as well, if you're using the `mayastor/mayastor` chart and not the `openebs/openebs` chart: `kubectl mayastor upgrade --set agents.core.rebuild.partial.enabled=false`
 :::
 
 ```
-helm upgrade openebs openebs/openebs -n openebs -f old-values.yaml --version 4.1.0 \
-  --set mayastor.agents.core.rebuild.partial.enabled=false \
```

Comment:
why is this not needed?

Reply:
Because upgrades from 3.x to 4.x require the flag `--set mayastor.agents.core.rebuild.partial.enabled=false`. Upgrades from 4.x to newer versions don't require it.

Do you think this is a better idea?
```
# Add the option --set mayastor.agents.core.rebuild.partial.enabled=false if the source version is a 3.x release.
helm upgrade openebs openebs/openebs -n openebs -f old-values.yaml --version 4.1.1 \
    --set openebs-crds.csi.volumeSnapshots.enabled=false

Reply:
LGTM